05-Time-Series-Raster.Rmd

```diff
@@ -92,102 +93,119 @@ This lesson is a part of a series of raster data in R lessons:
+images for each time period that NDVI is available.
 
 
-##Understanding the metadata
-?? we could teach this but Leah would need to rename the files to the original
-names -- thoughts?
-
-##Goals
-
-In this lesson, we will
-
-1. Import NDVI data derived from the Landsat Sensor in raster (`geotiff`) format
-2. Plot one full year of NDVI raster time series data. 
-3. Generate an average NDVI value for each time period throughout the year.
-
 <figure>
-    <a href="{{ site.baseurl }}/images/raster_timeseries/GreenessOverTime.png">
-    <img src="{{ site.baseurl }}/images/raster_timeseries/GreenessOverTime.png"></a>
-    <figcaption>A raster dataset can also contain a time series. In R, a stack of rasters 
-    will be in the same extent, CRS and resolution.</figcaption>
+    <a href="{{ site.baseurl }}/images/raster_timeseries/RGBSTack_1.png">
+    <img src="{{ site.baseurl }}/images/raster_timeseries/RGBSTack_1.png"></a>
+    <figcaption>A color image consists of 3 bands - red, green and blue. When
+    rendered together in a GIS, or even a tool like Photoshop or any other
+    image software, they create a color image. Image Source: NEON,Inc.  
+    </figcaption>
 </figure>
 
+
 ##Getting Started 
+In this lesson, we will use the `raster` and `rgdal` packages.
+
+```{r load-libraries }
+
+library(raster)
+library(rgdal)
 
-To begin, we will create a list of raster file names that can be used to
-generate a `RasterStack`. We can use `list.files` to generate the list. We will 
-tell R to only find files with a `.tif` extention using the syntax `pattern=".tif$"`.
+```
 
-If we specify `full.names=TRUE`, the full path for each file will be added to the 
-list. We will be able to create a `RasterStack` directly from the list.
+To begin, we will create a list of raster files that can be used to
+generate a `RasterStack`. We can use `list.files()` to designate the list. We
+will tell `R` to only find files with a `.tif` extention using the syntax
+`pattern=".tif$"`.
 
-Note that the full path is relative to the working directory that was set.
-{: .notice }
+If we specify `full.names=TRUE`, the full path for each file will be added to
+the list. We will be able to create a `RasterStack` directly from the list.
 
 ```{r import-ndvi-rasters }
 
 # Create list of NDVI file paths
-NDVI_path <- "Landsat_NDVI/HARV/2011/ndvi"
+NDVI_path <- "Landsat_NDVI/HARV/2011/ndvi"  #assign path to object = cleaner code
 all_NDVI <- list.files(NDVI_path, full.names = TRUE, pattern = ".tif$")
 
-#view list - note that the full path (relative to our working directory)
-#is included
+#view list - note the full path, relative to our working directory, is included
 all_NDVI
 
 ```
 
-Now we have a list of all `geotiff` files in the `ndvi` directory for Harvard Forest.
-Once we have this list, we can create a stack and begin to work with the
+Now we have a list of all GeoTIFF files in the `ndvi` directory for Harvard
```

Comment:
I typically code comment file types as well so it stands out. This could be a topic added to our template repo.